force-app/main/default/classes/ServiceScheduleSelector_TEST.cls

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *  * Copyright (c) 2022, salesforce.com, inc.
+ *  * All rights reserved.
+ *  * SPDX-License-Identifier: BSD-3-Clause
+ *  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ *
+ */
+
+@IsTest
+public with sharing class ServiceScheduleSelector_TEST {
+    @TestSetup
+    private static void createScheduleData() {
+        TestDataFactory.generateServiceData();
+        TestDataFactory.generateServiceScheduleData('Daily', System.today());
+    }
+
+    @IsTest
+    private static void testGetServiceScheduleById() {
+        ServiceSchedule__c expectedSchedule = [
+            SELECT
+                Id,
```

Comment:
If we are only referencing the Id and the Name in the test, do we need to query the rest of the fields?